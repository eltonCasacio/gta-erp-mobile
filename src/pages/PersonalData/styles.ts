import styled from 'styled-components/native';
import Theme from '../../styles/Theme';

export const Wrapper = styled.View`
  flex: 1;
  justify-content: space-between;
`;

export const InfoWrapper = styled.View`
  flex: 1;
  margin: ${Theme.Spacings.medium} 0;
  background-color: ${Theme.Colors.white};
`;

export const Text = styled.Text`
  color: ${Theme.Colors.grayBold};
  font-size: ${Theme.Font.great};
  padding: ${Theme.Spacings.medium};
`;
