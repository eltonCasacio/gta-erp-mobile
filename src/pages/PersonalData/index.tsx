import React from 'react';

import * as S from './styles';

import Rodape from '../../components/Rodape';

type PersonalDataProps = {
  navigation: any;
};

const PersonalData = ({navigation}: PersonalDataProps) => {
  return (
    <S.Wrapper>
      <S.InfoWrapper>
        <S.Text>Informações do usuario</S.Text>
      </S.InfoWrapper>
      <Rodape navigation={navigation} />
    </S.Wrapper>
  );
};

export default PersonalData;
