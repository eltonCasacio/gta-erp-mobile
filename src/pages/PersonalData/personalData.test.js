import React from 'react';
import renderer from 'react-test-renderer';

import PersonalData from '.';

describe('<>', () => {
  it('should render default styles', () => {
    const tree = renderer.create(<PersonalData />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
