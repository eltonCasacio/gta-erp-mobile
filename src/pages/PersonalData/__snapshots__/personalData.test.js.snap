// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<> should render default styles 1`] = `
<View
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "space-between",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#fafafa",
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "marginBottom": 10,
          "marginLeft": 0,
          "marginRight": 0,
          "marginTop": 10,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#707070",
            "fontSize": 22,
            "paddingBottom": 10,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 10,
          },
        ]
      }
    >
      Informações do usuario
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "flexDirection": "row",
          "height": 100,
          "justifyContent": "space-around",
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <RCTScrollView
        horizontal={true}
        showsHorizontalScrollIndicator={false}
        style={
          Array [
            Object {},
          ]
        }
      >
        <View>
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#5762F2",
                "borderRadius": 5,
                "height": 100,
                "justifyContent": "center",
                "marginBottom": 2,
                "marginLeft": 2,
                "marginRight": 2,
                "marginTop": 2,
                "opacity": 1,
                "paddingBottom": 5,
                "paddingLeft": 5,
                "paddingRight": 5,
                "paddingTop": 5,
                "width": 85,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#fafafa",
                    "fontSize": 16,
                    "textAlign": "center",
                  },
                ]
              }
            >
              Registrar Ponto
            </Text>
          </View>
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#5762F2",
                "borderRadius": 5,
                "height": 100,
                "justifyContent": "center",
                "marginBottom": 2,
                "marginLeft": 2,
                "marginRight": 2,
                "marginTop": 2,
                "opacity": 1,
                "paddingBottom": 5,
                "paddingLeft": 5,
                "paddingRight": 5,
                "paddingTop": 5,
                "width": 85,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#fafafa",
                    "fontSize": 16,
                    "textAlign": "center",
                  },
                ]
              }
            >
              Download Holerite
            </Text>
          </View>
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#5762F2",
                "borderRadius": 5,
                "height": 100,
                "justifyContent": "center",
                "marginBottom": 2,
                "marginLeft": 2,
                "marginRight": 2,
                "marginTop": 2,
                "opacity": 1,
                "paddingBottom": 5,
                "paddingLeft": 5,
                "paddingRight": 5,
                "paddingTop": 5,
                "width": 85,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#fafafa",
                    "fontSize": 16,
                    "textAlign": "center",
                  },
                ]
              }
            >
              Relatório
            </Text>
          </View>
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#5762F2",
                "borderRadius": 5,
                "height": 100,
                "justifyContent": "center",
                "marginBottom": 2,
                "marginLeft": 2,
                "marginRight": 2,
                "marginTop": 2,
                "opacity": 1,
                "paddingBottom": 5,
                "paddingLeft": 5,
                "paddingRight": 5,
                "paddingTop": 5,
                "width": 85,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#fafafa",
                    "fontSize": 16,
                    "textAlign": "center",
                  },
                ]
              }
            >
              Inf. Empresa
            </Text>
          </View>
        </View>
      </RCTScrollView>
    </View>
  </View>
</View>
`;
